ssas/service/admin/api.go

```diff
@@ -34,6 +34,8 @@ import (
 		500: serverError
 */
 func createGroup(w http.ResponseWriter, r *http.Request) {
+	ssas.Logger.Infof("Raw request createGroup: %v", r.Body)
```

Comment:
The resulting format in the logs is a bit rough; running `make test` yields the following line:
```
{"level":"info","msg":"Raw request updateGroup: \u0026{0xc0001bd960 \u003cnil\u003e \u003cnil\u003e false true {0 0} false false false 0x6b5f80}","time":"2020-02-10T15:10:02.755586Z"}
```

I'm thinking the `http.Request.Body` needs some special treatment as an `io.ReadCloser`.